backtrack/factor_combinations.py

```diff
@@ -42,8 +42,8 @@ def getFactors(self, n):
         n, i, combi = todo.pop()
         while i * i <= n:
             if n % i == 0:
-                combis += combi + [i, n/i],
-                todo += (n/i, i, combi+[i]),
+                combis.append(combi + [i, n/i]),
+                todo.append(n/i, i, combi+[i]),
```

Comment:
@yunshuipiao todo.append((n/i, i, combi+[i])) instead of todo.append(n/i, i, combi+[i]). Closing parenthesis is missing. append() can only accept a one parameter.

Reply:
Please create a new pull request. Thanks.

Reply:
sorry， I will fix it。